diff --git a/node_modules/@uniswap/sdk/dist/constants.d.ts b/node_modules/@uniswap/sdk/dist/constants.d.ts
index 65a9f9b..40796a4 100644
--- a/node_modules/@uniswap/sdk/dist/constants.d.ts
+++ b/node_modules/@uniswap/sdk/dist/constants.d.ts
@@ -1,11 +1,11 @@
 import JSBI from 'jsbi';
 export declare type BigintIsh = JSBI | bigint | string;
 export declare enum ChainId {
-    MAINNET = 1,
-    ROPSTEN = 3,
-    RINKEBY = 4,
-    GÖRLI = 5,
-    KOVAN = 42
+    MAINNET = 56,
+    ROPSTEN = 97,
+    RINKEBY = 97,
+    GÖRLI = 97,
+    KOVAN = 97
 }
 export declare enum TradeType {
     EXACT_INPUT = 0,
diff --git a/node_modules/@uniswap/sdk/dist/entities/token.d.ts b/node_modules/@uniswap/sdk/dist/entities/token.d.ts
index 3b5b50b..c9a88b1 100644
--- a/node_modules/@uniswap/sdk/dist/entities/token.d.ts
+++ b/node_modules/@uniswap/sdk/dist/entities/token.d.ts
@@ -25,9 +25,6 @@ export declare class Token extends Currency {
  */
 export declare function currencyEquals(currencyA: Currency, currencyB: Currency): boolean;
 export declare const WETH: {
-    1: Token;
-    3: Token;
-    4: Token;
-    5: Token;
-    42: Token;
+    56: Token;
+    97: Token;
 };
